### ISSUE TITLE
Add man pages from jack2 repo

### ISSUE THREAD

**Issue by dvzrv** [jackaudio/jack-example-tools#4]
The jack2 repo tracks [a bunch of man pages](https://github.com/jackaudio/jack2/tree/93619ed0c528e0f1e30513505bc0e8611c3ec2e8/man) for the tools/example-clients.

These should be included in this repository and be installed alongside the binaries.

**Comment by falkTX** [comment]
Is it possible to import only the `man` folder from the jack2 repo?
We can then delete the unused stuff after the import.

**Comment by dvzrv** [comment]
yes, it can be achieved with git-filter-repo as well. Just not sure yet if this can happen in a pull request or if I have to push straight to the default branch for this.
